packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -388,8 +539,9 @@ class FixedExtentScrollPhysics extends ScrollPhysics {
 /// The children are rendered as if rotating on a wheel instead of scrolling on
 /// a plane.
 class ListWheelScrollView extends StatefulWidget {
-  /// Creates a box in which children are scrolled on a wheel.
-  const ListWheelScrollView({
```

Comment:
This is an API break which is ok, but please follow the process at https://flutter.io/design-principles/#handling-breaking-changes